docs/compilers/CSharp/Compiler Breaking Changes - DotNet 8.md

```diff
@@ -0,0 +1,28 @@
+# This document lists known breaking changes in Roslyn after .NET 7 all the way to .NET 8.
+
+## Inferred delegate type for methods with default parameter values or `params` arrays is no longer `System.Action` or `System.Func`
+
+***Introduced in Visual Studio 2022 version 17.5***
+
+In C# 11 or earlier, methods with default parameter values or `params` arrays
+are inferred as `System.Action` or `System.Func` as demonstrated in the following code:
```

Comment:
Perhaps "delegate types inferred from methods ignored default parameter values and `params`"